examples/Test Sequencer/generic dut/sequencer_generic_dut.py

```diff
@@ -0,0 +1,147 @@
+    DUT commands, and test times.
+
+    To use this example with your DUT or other external hardware,
+    replace the 'dut_control' function with your specific DUT's API
+    or communication methods, as needed.
+    """
+    print("\nLaunching FlexLogger...")
+    with Application.launch() as app:
+        print("FlexLogger Launched Successfully!\n")
+        dut_control("powerOn")
+        dut_control("updateFirmware")
+        test1(app, config_file_path)
+        dut_control("powerOff")
+    print("FlexLogger Closed Successfully")
+    return
+
+
+def test1(app_reference, config_path):
+    if not os.path.isabs(config_path):
+        config_path = os.path.abspath(config_path)
+    with open(config_path, newline="") as csvfile:
+        csvdata = csv.reader(csvfile)
+        next(csvdata)  # skip header
+        for row in csvdata:
+            [
+                path,
+                state0,
+                time0,
+                state1,
+                time1,
+                state2,
+                time2,
+                state3,
+                time3,
+            ] = row
+            if not os.path.isabs(path):
+                config_root_dir = os.path.split(config_path)[0]
+                path = os.path.normpath(os.path.join(config_root_dir, path))
+            print("Loading Project...")
+            try:
+                project = app_reference.open_project(path)
+            except FlexLoggerError:
+                if not os.path.isfile(path):
+                    raise FileNotFoundError(
+                        "\n\nCheck CSV file. Cannot find FlexLogger project at path:\n{}".format(
+                            path
+                        )
+                    )
+                else:
+                    raise
+            print("Project Loaded\n")
+            test_session = project.test_session
+            print("Starting Test...")
+            start_test(test_session)
+            print("Test Started\n")
+            dut_control("changeState", state0)
+            display_elapsed_test_time(time0, test_session)
+            dut_control("changeState", state1)
+            display_elapsed_test_time(time1, test_session)
+            print("Pause Test")
+            test_session.pause()
+            dut_control("changeState", state2)
+            time.sleep(5)
+            print("Resume Test")
+            test_session.resume()
+            display_elapsed_test_time(time2, test_session)
+            dut_control("changeState", state3)
+            display_elapsed_test_time(time3, test_session)
+            test_session.stop()
+            print("Test Completed.\n\nClosing Project...")
+            project.close()
+            print("Project Closed\n")
+    return
+
+
+def start_test(
+    test_session, retries=3
+):  # This is a workaround for the FlexLogger API .start() method bug
+    for retry in range(retries):
+        test_session.start()
+        if test_session.state.value == 2:  # Running
```

Comment:
I think this can be
```python
if test_session.state == TestSessionState.RUNNING:
```
instead, which is a bit more readable. (you'd also need to import `TestSessionState` from `flexlogger.automation` as well)

Reply:
Agreed. I thought of this too. Kept the match simpler with the in comparison, but I agree this is more readable. Updated.